src/main/scala/example/TestHarness.scala

```diff
@@ -15,13 +15,27 @@ class TestHarness(implicit val p: Parameters) extends Module {
   val dut = p(BuildTop)(clock, reset.toBool, p)
   dut.debug := DontCare
   dut.connectSimAXIMem()
+  dut.connectSimAXIMMIO()
   dut.dontTouchPorts()
   dut.tieOffInterrupts()
+  dut.l2_frontend_bus_axi4.foreach(axi => {
+    axi.tieoff()
+    experimental.DataMirror.directionOf(axi.ar.ready) match {
+      case core.ActualDirection.Input =>
+        axi.r.bits := 0.U.asTypeOf(axi.r.bits)
+        axi.b.bits := 0.U.asTypeOf(axi.b.bits)
+      case core.ActualDirection.Output =>
+        axi.aw.bits := 0.U.asTypeOf(axi.aw.bits)
+        axi.ar.bits := 0.U.asTypeOf(axi.ar.bits)
+        axi.w.bits := 0.U.asTypeOf(axi.w.bits)
```

Comment:
Should this be `axi.w.bits := DontCare` ?
...etc

Reply:
Good point, I think that's better

Reply:
Any particular reason that AW, AR, W are handled differently than R & B?

Reply:
Because I'm dumb lol